suspect/processing/channel_combination.py

```diff
@@ -19,3 +24,125 @@ def combine_channels(data, weights=None):
     weighted_data = weights.reshape((len(weights), 1)) * data
     combined_data = weighted_data.sum(axis=0)
     return combined_data
+    
+
+def RAMRS_SVD_weighting(data, phase_correct = True, whiten = False, W = None, noise_range = None):
```

Comment:
Whitening the channel noise is something that I have been meaning to do for a long time, so very pleased to see that has made an appearance. However, I am not sure this is the best way to set up the interface. My general approach throughout has been to provide small, modular functions to perform each step of the processing, rather than to give a few monolithic functions with lots of options in the parameter string, like whiten, phase_correct etc. This is easier to understand, easier to test and easier to customise. I would much prefer code like
```
whitened_wref = whiten(wref_data)
channel_weights = svd_weighting(whitened_wref)
combined_data = np.average(wsup_data, axis=1, weights=channel_weights)
```
rather than 
```
combined_data = RAMRS_SVD_weighting(wsup_data, phase_correct=True, whiten=True, W = channel_weights)
```